pkg/engine/armvariables_test.go

```diff
@@ -119,13 +119,13 @@ func TestK8sVars(t *testing.T) {
 		"mountetcdScript":                           GetKubernetesB64Mountetcd(),
 		"nsgID":                                     "[resourceId('Microsoft.Network/networkSecurityGroups',variables('nsgName'))]",
 		"nsgName":                                   "[concat(variables('masterVMNamePrefix'), 'nsg')]",
-		"orchestratorNameVersionTag":                "Kubernetes:1.11.8",
+		"orchestratorNameVersionTag":                "Kubernetes:1.11.9",
```

Comment:
are we going to have to update these tests every time there's a new default k8s version patch? Is there any way we can make them expect the latest patch instead of hardcoding to a specific patch?

Reply:
I'm sure there is a way. I'll enter a ticket to refactor these tests so we're not chasing 1.11.x patches.